docs/buildah-copy.md

```diff
@@ -21,6 +21,10 @@ Defaults to false.
 Note: You can also override the default value of --add-history by setting the
 BUILDAH\_HISTORY environment variable. `export BUILDAH_HISTORY=true`
 
+**--chmod** *permissions*
+
+Sets the access permissions of the destination content. Accepts the numerical format.
```

Comment:
It would be nice to add an example in the example section here and on the add man page.

Reply:
Sure, will open a PR to complement tomorrow.